LICENSE-3rdparty.csv

```diff
@@ -209,10 +210,10 @@ core,"github.com/coreos/go-systemd/sdjournal",Apache-2.0
 core,"github.com/coreos/go-systemd/v22/dbus",Apache-2.0
 core,"github.com/coreos/pkg/capnslog",Apache-2.0
 core,"github.com/coreos/pkg/dlopen",Apache-2.0
-core,"github.com/cyphar/filepath-securejoin",NewBSD
+core,"github.com/cyphar/filepath-securejoin",BSD-3-Clause
 core,"github.com/davecgh/go-spew/spew",ISC
 core,"github.com/dgraph-io/ristretto",Apache-2.0
-core,"github.com/dgraph-io/ristretto/z",Apache-2.0
+core,"github.com/dgraph-io/ristretto/z",MIT
```

Comment:
This is a very tricky case since different parts of the code of that module have different licenses so I don't know if there is a right answer but I wanted to highlight it since it has changed